driver/src/price_finding/naive_solver.rs

```diff
@@ -17,16 +17,25 @@ pub enum OrderPairType {
 }
 
 trait Matchable {
+    /// Returns whether the orders can be matched. For this the tokens need
+    /// to match, there must be a price that satisfies both orders and if there
+    /// is a fee then one of the tokens must be the fee token.
     fn attracts(&self, other: &Order, fee: &Option<Fee>) -> bool;
+    /// Returns whether the account to which the order belongs to has at least
+    /// as many funds of the sell token as the order's sell amount.
     fn sufficient_seller_funds(&self, state: &AccountState) -> bool;
     fn match_compare(
```

Comment:
Are we missing a doc string for this?

Reply:
I wasn't able to summarize this in a meaningful way in a single sentence so I skipped it. The reason I added the docstrings is that it helped me understand the code when I first read and I thought they can be useful to other people in the future. I don't think its necessary to docstring everything in this PR.

Reply:
Good enough. I totally understand!